website/src/blogs/storage-policies-its-different-this-time.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Storage Policies — It’s different this time
+author: Amit Kumar Das
+author_info: Engineer the DAO
+date: 29-11-2017
+tags: Container, DevOps, Kubernetes, Storage Policy, Updates
+excerpt: One of the most common disbelief at the operator’s end would be the reports of an application’s (that consumed this storage) sudden death after introducing a much awaited shiny new storage feature.
+---
+
+
+### Need for Storage Policy
+
+One of the most common disbelief at the operator’s end would be the reports of an application’s (*that consumed this storage*) sudden death after introducing a much awaited shiny new storage feature. To make things worse the same surprise would be reciprocated from the storage provider as well. Neither the operator nor the storage fellow have any clues to this sudden strangeness.
+
+Having gone through many such cycles of frustrations building storage features that satisfy the needs of every operator as well does not break stuff that are outside its control; I knew there was clearly a gap in storage design that needs to be talked about openly and bridged. In this article, I present storage policy as the solution to this pressing problem.
+
+The moment I think about conceptualizing storage policy, my mind tries to battle against many odds. *Is this not what everyone (read storage vendors) has accomplished?*
+
+**Think again!!!** The chances are, it works for a specific storage **“A”** and needs a radically different approach for the other storage **“B”**; assuming both of these provide the infrastructure for the operator’s storage needs. So the operator goes ahead and does some plumbings if lucky enough to get these infrastructures as white-box components. *At the end of the day the operator is ready with a new bespoke program and feels so lucky*. Whatsoever, this does not last long due the very behaviour of infra components. *This operator needs to be ready all the time for the next upgrade to above crafted program,**since little can be expected from the monolith storage fellas*. *Poor operator is left to fend for self, building all possible defence mechanisms*.
```

Comment:
```suggestion
**Think again!!!** The chances are, it works for specific storage **“A”** and needs a radically different approach for the other storage **“B”**; assuming both of these provide the infrastructure for the operator’s storage needs. So the operator goes ahead and does some plumbings if lucky enough to get these infrastructures as white-box components. *At the end of the day, the operator is ready with a new bespoke program and feels so lucky*. Whatsoever, this does not last long due to the very behaviour of infra components. *This operator needs to be ready all the time for the next upgrade to above crafted program, **since little can be expected from the monolith storage fellas*. *Poor operator is left to fend for self, building all possible defence mechanisms*.
```